sqe-realtime-hub-builder/Program.cs

```diff
@@ -69,24 +94,34 @@ private static void ParseSqeHttpControllers()
             var dir = new DirectoryInfo(controllerFolder);
             var files = dir.GetFiles("*.cs");
             var classFields = new List<ControllerField>();
+            var hubInterfaces = new List<string>();
 
             // Delete any existing Realtime Hubs
             Console.WriteLine($"Deleting any existing realtime hub methods from {hubFolder}.");
             var folderInfo = new DirectoryInfo(hubFolder);
-            foreach (var file in folderInfo.GetFiles()) file.Delete();
+            foreach (var file in folderInfo.GetFiles())
+                if (file.Name != "HubInterface.cs")
```

Comment:
I'm a little worried about a directory that includes both auto generated *and* manually written code files. Perhaps the auto generated files should all go into the `AutoGenerated` subdirectory (they can still use the same namespace, so nobody else notices it)?